Signal/src/views/ContactTableViewCell.m

```diff
@@ -192,10 +194,11 @@ - (void)layoutSubviews
         OWSAssert(self.nameLabel.superview == self.contentView);
         const CGFloat kSubtitleVMargin
             = ((self.contentView.height - self.nameLabel.font.lineHeight) * 0.5f - self.subtitle.height) * 0.5f;
-        self.subtitle.frame = CGRectMake(self.nameLabel.left,
-            round((self.contentView.height - self.subtitle.height) - kSubtitleVMargin),
-            self.subtitle.width,
-            self.subtitle.height);
+        self.subtitle.frame
```

Comment:
I was doing custom layout to get the vertical positioning exactly right.  But you're right, there was a simpler way.  Thanks.